### PR TITLE
WAITP-1239: Fixes for static text visuals

### PR DESCRIPTION
**Pull request by acdunham**
### Changes:
- Fix differences in static text visuals
- Fix issue where url for download link is incorrect if there is no trailing `/`

### CODE REVIEW

File: packages/tupaia-web/src/utils/transformDownloadLink.ts [hunk 1]
```diff
@@ -10,5 +10,6 @@ import { API_URL } from '../api';
 export const transformDownloadLink = (resourceUrl: string) => {
   const baseUrl = API_URL;
   if (resourceUrl.includes('http')) return resourceUrl;
-  return `${baseUrl}${resourceUrl}`;
+  if (baseUrl.endsWith('/')) return `${baseUrl}${resourceUrl}`; // remove trailing slash from baseUrl if it exists
```
Comment (tcaiger) [inline packages/tupaia-web/src/utils/transformDownloadLink.ts:13]: I think that [trim()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trim) would do the trick here, right?

Comment (acdunham) [reply packages/tupaia-web/src/utils/transformDownloadLink.ts:13]: That would only work with whitespace right?

Comment (tcaiger) [reply packages/tupaia-web/src/utils/transformDownloadLink.ts:13]: Oh yeah sorry - you are totally correct - I didn't read it correctly.

**Review by tcaiger — APPROVED** [review]
Thanks for the fiddly little fix ups

**Pull request merged**